response_test.go

```diff
@@ -0,0 +1,286 @@
+package amazonpa
+
+import (
+	"encoding/xml"
+	"testing"
+)
+
+func assertEqualStr(t *testing.T, v1, v2 string, msg ...string) {
+	if v1 != v2 {
+		t.Error(msg)
+	}
+}
+
+func assertEqualInt(t *testing.T, v1, v2 int, msg ...string) {
+	if v1 != v2 {
+		t.Error(msg)
+	}
+}
+
+func assertEqualFloat(t *testing.T, v1, v2 float64, msg ...string) {
+	if v1 != v2 {
+		t.Error(msg)
+	}
+}
+
+func assertEqualBool(t *testing.T, v1, v2 bool, msg ...string) {
+	if v1 != v2 {
+		t.Error(msg)
+	}
+}
+
+func TestParseItemLookupResponse(t *testing.T) {
+	const responseData = `
```

Comment:
Maybe is better to put this in a `testdata` directory and not here in the source.